src/services/chain/nft/baseledger-staking-og/baseledger-staking-og.ts

```diff
@@ -0,0 +1,116 @@
+import * as Sentry from '@sentry/vue';
+import Web3 from 'web3';
+import { TransactionReceipt } from 'web3-eth';
+import { ContractSendMethod } from 'web3-eth-contract';
+import { AbiItem } from 'web3-utils';
+
+import { floorDivide, multiply } from '@/utils/bigmath';
+import { Network } from '@/utils/networkTypes';
+import {
+  lookupAddress,
+  NFT_BASELEDGER_STAKING_OG_ABI
+} from '@/wallet/references/data';
+import { TransactionsParams } from '@/wallet/types';
+
+import { Step } from '@/components/forms/form-loader/types';
+
+import { BaseledgerStakingOGData } from './types';
+
+export const getBaseledgerStakingOGData = async (
+  accountAddress: string,
+  network: Network,
+  web3: Web3
+): Promise<BaseledgerStakingOGData> => {
+  const transactionParams = {
+    from: accountAddress
+  } as TransactionsParams;
+
+  const contractAddress = lookupAddress(network, 'NFT_BASELEDGER_STAKING_OG');
+
+  if (contractAddress === '0x1') {
```

Comment:
```suggestion
  if (isDefaultAddress(contractAddress)) {
```
or
```suggestion
  if (contractAddress === defaultAddress) {
```

`defaultAddress` string or `isDefaultAddress` predicate is an exported member of `references/data.ts` (at least it is in some feature branches, let's check if it's available in this context)